src/Dialect/ONNX/DialectBuilder.cpp

```diff
@@ -28,6 +28,11 @@ namespace onnx_mlir {
 
 //====-------------------------- ONNX Builder ---------------------------===//
 
+IntegerAttr OnnxBuilder::getSignedInt64Attr(int64_t n) const {
+  return IntegerAttr::get(b().getIntegerType(64, /*isSigned=*/true),
+      APInt(64, n, /*isSigned=*/true));
```

Comment:
you can just pass n as 2nd arg to IntegerAttr::get, no need for APInt

Reply:
nice, thanks.